src/runtime_src/core/edge/drm/zocl/zocl_rpu_channel.c

```diff
@@ -389,7 +414,7 @@ static void zchan_cmd_handler(struct platform_device *pdev, struct xgq_cmd_sq_hd
 	cmd_handler func = opcode2handler(op);
 	struct xgq_com_queue_entry r = {};
 
-	zchan_info(chan, "%s received", opcode2name(op));
+	//	zchan_info(chan, "%s received", opcode2name(op));
```

Comment:
if no needed, please remove this log or switch to debug.

Reply:
Switched to zchan_dbg